native/asb/src/main/java/io/ballerinax/asb/listener/MessageListener.java

```diff
@@ -184,20 +228,19 @@ public Object stop(BObject listenerBObject) {
     }
 
     /**
-     * Stops consuming messages through all the consumer services and terminates the connection with server.
+     * Stops consuming messages through all the consumer services and terminates the
+     * connection with server.
      *
      * @param listenerBObject Ballerina listener object.
      * @return An error if listener fails to abort the connection.
      */
     public Object abortConnection(BObject listenerBObject) {
```

Comment:
At which point do we use this?

Reply:
This is used in listener when terminating the connection with the server.

Reply:
OK. I think it gracefully closes. +1